xml_converter/src/attribute/bool.cpp

```diff
@@ -21,3 +21,16 @@ bool parse_bool(rapidxml::xml_attribute<>* input, vector<XMLError*> *errors) {
         return false;
     }
 }
+
+string stringify_bool(bool attribute_value){
+    if (attribute_value == false) {
```

Comment:
We can just do "if" and "else" because  bool can only be true or false

Reply:
Done

Reply:
Lets actually change this to `if (attribute_value) { return "true" } else { return "false"}` (with proper spacing.

Reply:
Done